bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -35,226 +35,229 @@ Location,Garden,Outdoor,Garden,Gardens,
+Equipment,Gate,Door,Gate,Gates,Gate
+Equipment,InnerDoor,Door,Inner Door,Inner Doors,Inner door
+Equipment,SideDoor,Door,Side Door,Side Doors,Side door
+Equipment,Doorbell,,Doorbell,Doorbells,Doorbell
+Equipment,DrinkingWater,,Drinking Water,Potable Water,Related to drinking water
+Equipment,HotWaterFaucet,DrinkingWater,Hot Water Faucet,"Hot Water Tap, Boiling Water Tap, Boiling Water Faucet",Hot water faucet
+Equipment,WaterFilter,DrinkingWater,Water Filter,,Water filter
+Equipment,WaterSoftener,DrinkingWater,Water Softener,,Water softener
+Equipment,HVAC,,HVAC,"Heating, Ventilation, Air Conditioning, A/C, A/Cs, AC",Related to Heating Ventilation or AirConditioning
+Equipment,AirConditioner,HVAC,Air Conditioner,,Air conditioner
+Equipment,AirFilter,HVAC,Air Filter,,Air filter
+Equipment,Boiler,HVAC,Boiler,Boilers,Boiler
+Equipment,Dehumidifier,HVAC,Dehumidifier,,Dehumidifier
+Equipment,Fan,HVAC,Fan,Fans,Fan
+Equipment,CeilingFan,Fan,Ceiling Fan,Ceiling Fans,Ceiling fan
+Equipment,ExhaustFan,Fan,Exhaust Fan,"Extract Fan, Toilet Fan",Exhaust fan
+Equipment,KitchenHood,Fan,Kitchen Hood,Kitchen Hoods,Kitchen hood
+Equipment,FloorHeating,HVAC,Floor Heating,"Underfloor Heating, Radiant Floor Heating",Floor heating
+Equipment,Furnace,HVAC,Furnace,"Wood Burner, Wood Heater",Furnace
+Equipment,HeatPump,HVAC,Heat Pump,,Heat pump
+Equipment,HeatRecovery,HVAC,Heat Recovery,Energy Recovery,Heat recovery
+Equipment,Humidifier,HVAC,Humidifier,,Humidifier
+Equipment,RadiatorControl,HVAC,Radiator Control,"Radiator Controls, Radiator, Radiators",Radiator control
+Equipment,SmartVent,HVAC,Smart Vent,,Smart ventilation device
+Equipment,Thermostat,HVAC,Thermostat,,Thermostat
+Equipment,WaterHeater,HVAC,Water Heater,Water Boiler,Water heater
+Equipment,Horticulture,,Horticulture,,Related to horticulture
+Equipment,Irrigation,Horticulture,Irrigation,"Sprinkler, Drip System",Related to irrigation
+Equipment,LawnMower,Horticulture,Lawn Mower,Lawn Mowers,Lawn mower
+Equipment,SoilSensor,Horticulture,Soil Sensor,Moisture Sensor,Soil sensor
+Equipment,LightSource,,Light Source,"Lights, Lighting",Light source
+Equipment,AccentLight,LightSource,Accent Light,,Accent light
+Equipment,Chandelier,LightSource,Chandelier,,Chandelier
+Equipment,Downlight,LightSource,Downlight,"Can Light, Pot Light",Downlight
+Equipment,FloodLight,LightSource,Flood Light,,Flood light
+Equipment,Lamp,LightSource,Lamp,,Lamp
+Equipment,LightStrip,LightSource,Light Strip,LED Strip,Light strip
+Equipment,LightStripe,LightSource,Light Stripe,Light Stripes,Light stripe
+Equipment,Lightbulb,LightSource,Light Bulb,"Lightbulbs, Bulb, Bulbs, Lamp, Lamps",Light bulb
+Equipment,Pendant,LightSource,Pendant,,Pendant light
+Equipment,Sconce,LightSource,Sconce,,Sconce light
+Equipment,SpotLight,LightSource,Spot Light,,Spot light
+Equipment,TrackLight,LightSource,Track Light,,Track light
+Equipment,WallLight,LightSource,Wall Light,,Wall light
+Equipment,Lock,,Lock,Locks,Door- or window- lock
```

Comment:
```suggestion
Equipment,Lock,,Lock,Locks,
```
May apply to bike lock, car lock, software lock, (computer/mobile phone) screen lock, etc.